org.yamagi.YamagiQ2.appdata.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Kevin Degeling -->
+
+<component type="desktop">
+  <id>org.yamagi.YamagiQ2</id>
+  <name>Yamagi Quake II</name>
+  <summary>Yamagi Quake II is an alternative client for id Software's Quake II</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+
+  <developer_name>Yamagi team</developer_name>
+  <update_contact>gnome@kevindegeling.nl</update_contact>
+
+  <url type="homepage">https://www.yamagi.org/quake2//</url>
+  <url type="bugtracker">https://github.com/yquake2</url>
+  <url type="help">https://github.com/yquake2/yquake2/blob/master/doc/01_index.md</url>
+
+  <launchable type="desktop-id">io.github.ezQuake.desktop</launchable>
+
+  <description>
+    <p>
+     Yamagi Quake II is an alternative client for id Software's Quake II. Our goal is to provide the best Quake II experience possible, we strive to preserve the gameplay as it was back in 1997. Thus we aim mostly for bug fixes, stability and gentle enhancements were appropriate. Yamagi Quake II has a lot of unique features. The most notables ones are:
+    </p>
+    <ul>
+      <li>A modern OpenGL 3.2 renderer, providing a greatly enhanced look and feel. The original OpenGL 1.4 and even the Software renderers are still supported. Both 3D accelerated renderers support anisotropic filtering and multisampling anti aliasing.</li>
+      <li>Full support for widescreen displays and arbitrary resolutions. The HUD can be scaled to be readable on high resolutions. </li>
+      <li>Support for surround sound, from stereo up to 7.1 channels and even HRTF. The background music can be played from OGG/Vorbis files, a CD drive is no longer necessary.</li>
+      <li>A rewritten savegame system. Yamagi Quake II savegames are completely reliable and backward compatible between releases as long as the principle architecture stays the same.</li>
+      <li>Highly precise timings. For example, on a flat panel display with a 59.95 hertz refresh rate and vertical synchronisation (VSync) enabled Yamagi Quake II is running at exactly 59.95 frames per second.</li>
+      <li>In comparison with the original client over 1.500 bugs were fixed. The Yamagi Quake II client is very well tested and stable. The dedicated server is able to run for weeks without a single restart. Last but not least the code is fully 64 bit clean and highly portable.</li>
+    </ul>
+    <p>
+     Commercial data files are required to run the supported game. These can be aquired though a multitude of sources. See the manual for more info on this.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/org.yamagi.YamagiQ2/master/images/image_01.png</image>
```

Comment:
Use your own repository for screenshots & switch back to this once the repo is created.

Reply:
Updated.